client/extensions/woocommerce/app/order/order-customer/create.js

```diff
@@ -77,7 +95,11 @@ class OrderCustomerInfo extends Component {
 	};
 
 	renderBilling = ( address = {} ) => {
-		const { translate } = this.props;
+		const { countries, loadedLocations, translate } = this.props;
+		if ( ! loadedLocations ) {
+			return null;
```

Comment:
Do we want to show some sort of loading indicator while the countries are loading?